server/routes/red-flags.js

```diff
@@ -11,5 +11,9 @@ router.get('/red-flags/:id', redFlags.getSingleRedFlag);
 
 router.post('/red-flags', midware.isValid, redFlags.createRedFlag);
 
+router.patch('/red-flags/:id/location', midware.isValid, redFlags.editLocation);
+
+router.patch('/red-flags/:id/comment',midware.isValid, redFlags.editComment);
```

Comment:
Based on what you have in your validation middleware, are you saying `location` must be present before the user can update a comment?